.DataLogTool/datalogtool.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
not sure if straight up deleting this is right, but it a path specific to Ivan's laptop definitely shouldn't be in Hydrogen

Reply:
It'll be regenerated, so just leave it blank